src/components/accordion/index.njk

```diff
@@ -0,0 +1,27 @@
+---
+title: Accordion
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+
+
+## When to use this component 
+This component can be used to condense large sections of related content on a page. It allows a user to quickly scan a collated group of content titles.
+
+## When not to use this component
+Accordions should not be used if the content inside the accordion is critial to the user to be able to progress with their task.
+
+{{
+    patternlibExample("components/accordion/examples/accordion/index.njk")
```

Comment:
Need to update to use `patternlibExample({"path":"components/accordion/examples/accordion/index.njk"})`

Reply:
Good catch 👍